vector/build.gradle

```diff
@@ -268,6 +278,40 @@ android {
             }
             // signingConfig signingConfigs.release
         }
+
+        nightly {
+            applicationIdSuffix ".nightly"
```

Comment:
we'll also need to update the sygnal config to include the new package/sha

Reply:
Are you sure about that? (I will double check that I can receive Push with the nightly)

Reply:
For me it should be working as it's a suffix

Reply:
ah you're right! it's working locally for me, I thought we'd need to introduce the package into https://github.com/matrix-org/sygnal/blob/main/sygnal.yaml.sample#L203

I guess we're okay because the GCM api key is the same for nightly and debug (I thought firebase would validate package names, shas and api keys :thinking: )